### ISSUE TITLE
Stop handle from cancelling too easily?

### ISSUE THREAD

**Issue by jbmaxwell** [bradhowes/Joystick#4]
I'm noticing that if I move too quickly, the handle very easily cancels and returns to its home position. I have to wait a moment (around 0.5 seconds) to get the handle to stay with my thumb as I move it. My application doesn't require the user to hold the joystick at all times, so it gets a bit frustrating having to re-grab, and wait, for the handle. 
Is there a way to make this more tolerant—i.e., something I can tweak to make it harder for the handle to cancel like this? Thanks!

**Comment by bradhowes** [comment]
Hi. I'm not sure what could be causing this, though it *might* be possibly related to the frame/bounds of the base view. I'll play around with it to see what I can uncover or perhaps fix. If this is indeed the issue, then one way to workaround it would be to create a JoyStickView instance with a frame that is large enough to entirely hold the handle at the maximum travel distance.

Also, if you could share some code that reproduces the issue you are seeing that would be great. :

**Comment by jbmaxwell** [comment]
Ah, okay. I wondered if it might be something like that. I'll keep an eye on this issue, but can also give it a try with a larger base view. Thanks!

**Comment by jbmaxwell** [comment]
Hmm... yeah, it's clearly hitting touchesCancelled too easily. I've tried a larger base image, with a smaller ratio for the handle size, but it's the same problem. Very strange. I've also tried a large handle size, with no change. It sometimes seems a bit arbitrary—every once in a while I can grab it and move quickly, without issue. Very odd. It almost seems like some kind of refresh rate problem (though I don't see how that could be).

[UPDATE: Okay, I see that swipes are being registered in the SCNView behind the joystick. That's probably what's up... will report back if I verify this.]

[UPDATE 2: Yup. That's what it was. Sorry for the spam.]
